developer/testscripts/aatest_ds000114_fmri_localpipeline.m

```diff
@@ -0,0 +1,126 @@
+function aatest_ds000114_fmri_localpipeline(parameterfile, deleteprevious,wheretoprocess)
+
+% This script runs a basic modeling pipeline (preprocessing through secondlevel)
+% using ds000114 motor and line bisection tasks. The data should be dowloaded 
+% prior to running the script
+
+% -------------------------------------------------------------------------
+% init
+% -------------------------------------------------------------------------
+
+aap = aa_test_inittest(mfilename('fullpath'), parameterfile, deleteprevious, wheretoprocess);
+aap = aarecipe(['$AAHOME/developer/testscripts/aatest_ds000114_fmri_localpipeline_prep.xml']);
```

Comment:
You MUST use `aa_test_inittest` rather than `aarecipe`.
```matlab
aap = aa_test_inittest([mfilename('fullpath') '_prep'], parameterfile, deleteprevious, wheretoprocess);
```